esma.cmake

```diff
@@ -16,7 +16,11 @@ elseif (IS_DIRECTORY "${CMAKE_CURRENT_LIST_DIR}/@ecbuild")
 elseif (IS_DIRECTORY "${CMAKE_CURRENT_LIST_DIR}/ecbuild@")
   list (APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/ecbuild@/cmake")
 else ()
-  message (FATAL_ERROR "ecbuild subdir not found")
+  if (DEFINED ENV{CMAKE_PREFIX_PATH})
```

Comment:
Hmm. @tclune Any thoughts on this use of `ENV`. I guess like `BASEDIR` I always append to `CMAKE_PREFIX_PATH` with `-DCMAKE_PREFIX_PATH=...` on the cmake command line.

Reply:
Thanks for pointing this out, @mathomp4. These lines are not needed and I've just removed them in commit 2e7fa7a